self-tests.yml

```diff
@@ -644,6 +644,29 @@ buildvariants:
       - name: ".test"
       - name: test-db-auth
 
+  - name: ubuntu2004-container
+    display_name: Ubuntu 20.04 (Container)
+    run_on:
+      - evg-container
+    expansions:
+      goos: linux
+      goarch: amd64
+      IS_DOCKER: true
+      GOROOT: /usr/local/go
+      mongodb_url: https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu2004-5.0.14.tgz
+      decompress: tar zxvf
+    tasks:
+      - name: "dist-and-upload-clis"
+      - name: "dist-staging"
+      - name: "dist-unsigned"
+        activate: false
+      - name: ".smoke"
+      - name: ".test"
+      - name: "js-test"
+      - name: ".linter"
+      - name: "docker-cleanup"
```

Comment:
This one cannot run because `test-thirdparty-docker` requires Docker, which is not recursively available when it's already running in a Docker container.

Reply:
Ah, thought it was passing but you're right it's just [no-opping](https://spruce.mongodb.com/task/evergreen_ubuntu2004_container_docker_cleanup_patch_65b839a80f0a9cb9523f25bbdbae4a3b93d2ffe2_644c1354850e615d1b3adaa4_23_04_28_18_41_34/logs?execution=0&sortBy=STATUS&sortDir=ASC)